packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info.tsx

```diff
@@ -38,21 +39,30 @@ const Info = observer(({ handleSelect, item, list }: TInfo) => {
         },
     } = useStore();
     const [selected_tab, setSelectedTab] = React.useState<TSelectedTab>(TABS.DESCRIPTION);
+    const [selected_contract_type, setSelectedContractType] = React.useState(item);
     const { RISE_FALL_EQUAL, TURBOS, VANILLA } = TRADE_TYPES;
     const contract_types: TContractType[] | undefined = getContractTypes(list, item)?.filter(
         (i: { value: TContractType['value'] }) =>
             i.value !== RISE_FALL_EQUAL && i.value !== TURBOS.SHORT && i.value !== VANILLA.PUT
     );
-    const has_toggle_buttons = /accumulator|turboslong|vanilla|multiplier/i.test(item.value);
-    const should_show_video = /accumulator|turboslong|vanilla/i.test(item.value);
+    const has_toggle_buttons = /accumulator|turboslong|vanilla|multiplier/i.test(selected_contract_type.value);
+    const should_show_video = /accumulator|turboslong|vanilla/i.test(selected_contract_type.value);
     const is_description_tab_selected = selected_tab === TABS.DESCRIPTION;
     const is_glossary_tab_selected = selected_tab === TABS.GLOSSARY;
     const width = is_mobile ? '328' : '528';
     const scroll_bar_height = has_toggle_buttons ? '464px' : '560px';
+    const button_name = contract_types?.find(item => item.value === selected_contract_type.value)?.text;
+
     const onClickGlossary = (e?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => {
         clickAndKeyEventHandler(() => setSelectedTab(TABS.GLOSSARY), e);
     };
 
+    const is_unavailable = !!list[0].contract_categories?.find(
+        item => item.is_unavailable && item.contract_types.find(type => type.value === selected_contract_type.value)
+    );
+
+    const should_show_drop_down = contract_types && contract_types.length > 1;
```

Comment:
Will rename.
No, optional chaining won't work because here we are comparing with number. But i can wrap with `Number()` and it should pass, like this:
`const should_show_dropdown = Number(contract_types?.length) > 1;`